src/Table/index.tsx

```diff
@@ -1,49 +1,177 @@
-const defaultStyle = `color: var(--Black, #000);font-family: Roboto Mono;font-size: 16px;font-style: normal;text-align: left;font-weight: 400;line-height: normal;`
-
-const Table: React.FC<React.PropsWithChildren<TableProps>> = (props) => (
-  <Wrapper {...props}>
-    <TR key={Math.random().toString()} style={props.styles?.tr}>
-      {props.headers.map((header: string) => (
-        <TH key={Math.random().toString()} style={props.styles?.th || {}}>
-          {header}
-        </TH>
-      ))}
-    </TR>
-    {props.rows.map((row: RowValue[]) => (
-      <TR key={Math.random().toString()} style={props.styles?.tr}>
-        {row.map((value: RowValue) => (
-          <TD key={Math.random().toString()}>{value}</TD>
-        ))}
-      </TR>
-    ))}
-  </Wrapper>
+
+export const NameCell = ({ name, date }: { name: string; date: string }) => (
+  <Col>
+    <Row>
+      <PadlockIcon />
+      <Title>{name}</Title>
+    </Row>
+    <Row>
+      <ClockIcon />
+      <Date>{date}</Date>
+    </Row>
+  </Col>
+)
+
+export const StatusCell = ({
+  color = '#27847a',
+  Icon,
+  status,
+}: {
+  color?: string
+  Icon: React.FC
+  status: string
+}) => (
+  <Col>
+    <Icon />
+    <Status color={color}>{status}</Status>
+  </Col>
 )
 
+const Table: React.FC<React.PropsWithChildren<TableProps>> = (props) => {
+  return (
+    <Background>
+      <Wrapper {...props}>
+        <TR
+          color={'not-page'}
+          key={Math.random().toString()}
+          style={props.styles?.tr}
+        >
+          {props.headers.map((header: string) => (
+            <TH key={Math.random().toString()} style={props.styles?.th || {}}>
+              {header}
+            </TH>
+          ))}
+        </TR>
+        {props.rows.map((row: RowValue[]) => (
+          <TR key={Math.random().toString()} style={props.styles?.tr}>
+            {row.map((value: RowValue) => (
+              <TD key={Math.random().toString()}>{value}</TD>
+            ))}
+          </TR>
+        ))}
+      </Wrapper>
+    </Background>
+  )
+}
+
+const Title = styled('div')`
+  color: #6aa685;
+  font-family: Inter;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 0px; /* 0% */
+`
+
+const Status = styled('div')`
+  width: 90px;
+  background-color: ${(props) => props.color || '#27847a'};
+  color: #fff;
+  border-radius: 60px;
+
+  text-align: center;
+  font-family: Roboto;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 10px; /* 83.333% */
+`
+
+const Date = styled('div')`
+  color: #27847a;
+
+  font-family: Inter;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 10px; /* 83.333% */
+`
+
+const Row = styled('div')`
+  display: flex;
+  flex-direction: row;
+  gap: 5px;
+  align-items: center;
+`
+
+const Col = styled('div')`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+`
```

Comment:
There's quite a lot of hard coded values in the component here and I can't quite work out where they are all used. Are these just used for the Hii story?

Reply:
These are for the specific cell, that contain multiple elements. Please refer to the screenshots attached to this message. 

However, cells can take any HTML/JSX/React Component so I've moved into the component just to keep client cleaner. I think these could be abstracted to story only which I'm doing now, but was just making easier to test since I can export
<img width="259" alt="Screenshot 2024-03-04 at 9 45 32 am" src="https://github.com/digicatapult/ui-component-library/assets/11794402/03d16eef-e972-4d68-b0f2-c9a5d5b23ebb">
<img width="153" alt="Screenshot 2024-03-04 at 9 45 30 am" src="https://github.com/digicatapult/ui-component-library/assets/11794402/20032af6-9e17-49e3-b713-f6a18e80bbc3">

Reply:
Just abstracted away but in order to test it I did have to duplicate Hyproof unique components, as the export from stories did not work as expected.

Reply:
FYI I don't think these components are exported. As this is just styling they should probably exist firstly in the story and then they should be in the `sqnc-hyproof-client` repo. There are other issues here around using `px` font sizes and some other hard coded stuff but it's not so bad if it's in the client as that's a demo.

Reply:
this should be resolved and lives in the `__fixtures__` folder so we can test and assert this variant